### PR TITLE
KH-286: Allow location search to support Khmer and  English 

### PR DESCRIPTION
**Pull request by kazlaw**
Link to ticket [here](https://mekomsolutions.atlassian.net/browse/KH-286)

### CODE REVIEW

**Review by kdaud — COMMENTED** [review]
@kazlaw as per the ticket comments this PR is supposed to be directed to the confidential config.

File: configs/openmrs_config/locations/locations.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-d84e5c0a-ff31-4288-9718-7acff068e14b,,100102. Chambak_HC,100102. Chambak_HC,100102. ចំបក់_HC,100102. Chambak_HC,,Login Location,TRUE,,,,,,,,,,,,,,
-609033e0-097c-46ba-87a2-58c749171708,,100103. Ta Mao_HC,100103. Ta Mao_HC,100103. តាម៉ៅ_HC,100103. Ta Mao_HC,,Login Location,TRUE,,,,,,,,,,,,,,
-3e135aad-ab0d-447d-85f8-1421f2455a59,,100105. Khsach Andet_HC,100105. Khsach Andet_HC,100105. ខ្សាច់អណ្តែត_HC,100105. Khsach Andet_HC,,Login Location,TRUE,,,,,,,,,,,,,,
-ec030870-1d8f-457f-960b-3ede527841a4,,100106. Pongro_HC,100106. Pongro_HC,100106. ពង្រ_HC,100106. Pongro_HC,,Login Location,TRUE,,,,,,,,,,,,,,
\ No newline at end of file
+2e013076-ddf3-4f44-9a10-54fb6a8855f7,,100101. មន្ទីរពេទ្យបង្អែកឆ្លូង_RH / Chhlong_RH,100101. Chhlong_RH,100101. មន្ទីរពេទ្យបង្អែកឆ្លូង_RH,100101. Chhlong_RH,,Login Location,TRUE,,,,,,,,,,,,,,
+d84e5c0a-ff31-4288-9718-7acff068e14b,,100102. ចំបក់_HC / Chambak_HC,100102. Chambak_HC,100102. ចំបក់_HC,100102. Chambak_HC,,Login Location,TRUE,,,,,,,,,,,,,,
+609033e0-097c-46ba-87a2-58c749171708,,100103. តាម៉ៅ_HC / 100103. Ta Mao_HC,100103. Ta Mao_HC,100103. តាម៉ៅ_HC,100103. Ta Mao_HC,,Login Location,TRUE,,,,,,,,,,,,,,
```
Comment (kdaud) [inline configs/openmrs_config/locations/locations.csv:4]:
```suggestion
609033e0-097c-46ba-87a2-58c749171708,,100103. តាម៉ៅ_HC / Ta Mao_HC,100103. Ta Mao_HC,100103. តាម៉ៅ_HC,100103. Ta Mao_HC,,Login Location,TRUE,,,,,,,,,,,,,,
```

Comment (kazlaw) [reply configs/openmrs_config/locations/locations.csv]: This has been addressed @kdaud

**Review by kdaud — APPROVED** [review]
LGTM

**Pull request merged**